test/impl_test.py

```diff
@@ -4,15 +4,16 @@
 import logging
 import os
 import sys
+import json
 import time
 import unittest
 import shutil
 
 # local imports
 from biokbase.workspace.client import Workspace
+from AssemblyUtil.AssemblyUtilClient import AssemblyUtil
```

Comment:
AssemblyUtilClient is not imported successfully. try to rerun 'kb-sdk insteall AssemblyUtil'

Reply:
Forgot to git add